src/main/kotlin/no/nav/familie/ef/sak/beregning/barnetilsyn/BeløpsperiodeBarnetilsynDto.kt

```diff
@@ -0,0 +1,17 @@
+package no.nav.familie.ef.sak.beregning.barnetilsyn
+
+import no.nav.familie.ef.sak.felles.dto.Periode
+import java.math.BigDecimal
+
+data class BeløpsperiodeBarnetilsynDto(
+        val periode: Periode,
+        val beløp: BigDecimal,
+        val beregningsgrunnlag: BeregningsgrunnlagBarnetilsynDto,
+)
+
+data class BeregningsgrunnlagBarnetilsynDto(
+        val utgiftsbeløp: BigDecimal,
+        val kontantstøttebeløp: BigDecimal,
+        val tilleggsstønadsbeløp: BigDecimal,
```

Comment:
Og samme med disse, bare `kontantstøtte` og 'tilleggsstønad`

Reply:
Beholder beløp på disse hvis det er greit. 

Sparer meg for bittelitt hjernevirksomhet - sjekke type resonere meg fram til at det er et beløp, ikke en boolean (har kontantstøtte el. l). Sparer kanskje 10sek en god dag ;) (dårlige dager snakker vi ikke om)

Reply:
Helt greit 👍